### ISSUE TITLE
[enterprise-4.6] Issue in file service_mesh/v1x/installing-ossm.adoc

### ISSUE THREAD

**Issue by pamoedom** [openshift/openshift-docs#27514]
### Which section(s) is the issue in?

In both Mesh installation sections (v1 and v2):

https://docs.openshift.com/container-platform/4.6/service_mesh/v1x/installing-ossm.html#jaeger-operator-install-elasticsearch_installing-ossm-v1x
https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/installing-ossm.html#jaeger-operator-install-elasticsearch_installing-ossm

### What needs fixing?

The documentation states the following when installing the Elasticsearch component:

~~~
If you have already installed the Elasticsearch Operator as part of OpenShift cluster logging, you do not need to install the Elasticsearch Operator again. The Jaeger Operator will create the Elasticsearch instance using the installed Elasticsearch Operator.
~~~

Which is later in contradiction with this step:

~~~
6. On the Install Operator page, select the A specific namespace on the cluster option and then select openshift-operators-redhat from the menu.
~~~

Because the Logging [documentation](https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-deploying.html#cluster-logging-deploy-console_cluster-logging-deploying) states this, which is the correct configuration:

~~~
c. Ensure that the All namespaces on the cluster is selected under Installation Mode.
~~~

If an specific namespace is used, as stated by Mesh documentation, the Jaeger production setting that can be set [here](https://docs.openshift.com/container-platform/4.6/service_mesh/v2x/upgrading-ossm.html#ossm-upgrading-config-tracing_upgrading-ossm) will not work as expected in order to use Elasticsearch as storage.

In summary, the Mesh docs regarding Elasticsearch installation should be fixed as follows, for example:

~~~
6. On the Install Operator page, select All namespaces on the cluster option and ensure that openshift-operators-redhat is selected under Installed Namespace.
~~~

Best Regards.

**Comment by pamoedom** [comment]
Please see also related https://github.com/openshift/openshift-docs/issues/27515

**Comment by JStickler** [comment]
Closed via #27568
